js/state/downloadUtil.js

```diff
@@ -83,6 +83,16 @@ const getDownloadItems = (state) => {
     .map((download, downloadId) => downloadId)
 }
 
+const getActiveDownloads = (state) => {
+  if (!state.get('downloads')) {
```

Comment:
Can we do a single `const downloads = state.get('downloads')` so we don't need to make 2 calls to `state.get`?

Reply:
done